supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -67,3 +67,834 @@ For documentation questions, contact rhel-docs@redhat.com.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:kerberos[kerberos]
+
+[discrete]
+[[hidden-replica]]
+==== image:images/yes.png[yes] hidden replica (noun)
+*Description*: A hidden replica is an IdM replica that has all services running and available, but its server roles are disabled, and clients cannot discover the replica because it has no SRV records in DNS.
+
+Hidden replicas are primarily designed for services such as backups, bulk importing and exporting, or actions that require shutting down IdM services. Since no clients use a hidden replica, administrators can temporarily shut down the services on this host without affecting any clients. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/planning_identity_management/planning-the-replica-topology_{ProjectNameID}#the-hidden-replica-mode_planning-the-replica-topology[The hidden replica mode].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:dns-srv-records[dns-srv-records]
+
+[discrete]
+[[host-system]]
+==== image:images/yes.png[yes] host system (noun)
+*Description*: The system on which the instrumentation modules (from SystemTap scripts) are compiled, to be loaded on target systems.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:target-system[target-system]
+
+[discrete]
+[[http-server]]
+==== image:images/yes.png[yes] HTTP server (noun)
+*Description*: See xref:web-server[web-server]
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[id-mapping]]
+==== image:images/yes.png[yes] ID mapping (noun)
+*Description*: SSSD can use the SID of an AD user to algorithmically generate POSIX IDs in a process called _ID mapping_. ID mapping creates a map between SIDs in AD and IDs on Linux.
+
+* When SSSD detects a new AD domain, it assigns a range of available IDs to the new domain. Therefore, each AD domain has the same ID range on every SSSD client machine.
+* When an AD user logs in to an SSSD client machine for the first time, SSSD creates an entry for the user in the SSSD cache, including a UID based on the user's SID and the ID range for that domain.
+* Because the IDs for an AD user are generated in a consistent way from the same SID, the user has the same UID and GID when logging in to any Red Hat Enterprise Linux system.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:id-ranges[id-ranges], xref:sssd[sssd]
+
+[discrete]
+[[id-ranges]]
+==== image:images/yes.png[yes] ID ranges (noun)
+*Description*: An ID range is a range of ID numbers assigned to the IdM topology or a specific replica. You can use ID ranges to specify the valid range of UIDs and GIDs for new users, hosts and groups. ID ranges are used to avoid ID number conflicts. There are two distinct types of ID ranges in IdM:
+
+* _IdM ID range_
+
```

Comment:
This needs a "+" on this line if you want the below text to be indented properly with this. (Same for the second item below)

Or you can use the `term:: text` syntax instead

Reply:
+1